Gems/MotionMatching/Code/Source/FeatureMatrixMinMaxScaler.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/std/string/conversions.h>
+#include <AzFramework/IO/LocalFileIO.h>
+#include <Allocators.h>
+#include <limits>
+#include <FeatureMatrixMinMaxScaler.h>
+
+namespace EMotionFX::MotionMatching
+{
+    AZ_CLASS_ALLOCATOR_IMPL(MinMaxScaler, MotionMatchAllocator, 0)
```

Comment:
nitpick: missing a semicolon at the end of the line.

This causes a minor annoyance when using Visual Studio, because enter a newline right after this line would cause it to indent another level.
i.e
```
    AZ_CLASS_ALLOCATOR_IMPL(MinMaxScaler, MotionMatchAllocator, 0)

```
          ^
     Cursor is here and not aligned with the first 'A' in AZ_CLASS_ALLOCATOR_IMPL

Missing semicolon is why code like the following is incorrect indents: https://github.com/o3de/o3de/blob/f8379e7c24dacb03fe9faeadb4c046205ee8b545/Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/NodeFunctionGeneric.h#L213-L214